docs/guides/universal-profile/browser-extension/01-install-browser-extension.md

```diff
@@ -12,130 +12,197 @@ import TabItem from '@theme/TabItem';
 The UP Browser Extenstion is currently in the **development alpha** version. DO NOT use this in production!
 :::
 
-In this guide, you will learn how to download + install the **Universal Profile browser extension** on Chrome, Edge, Opera or Brave.
+This guide will teach you how to download and install the Universal Profile browser extension on Chrome, Edge, Opera, or Brave.
 
-## Download + Install the extension
+## Download and Unpack the Extension
 
-Click on one of the links below to download the extension, based on the browser you want to install it to.
+Click on link below to download the extension, based on the browser to which you want to install it.
 
 ### :inbox_tray: **[Download link](https://storage.googleapis.com/up-browser-extension/universalprofile-extension-v1.0.0-develop.143.zip)**
 
+Open your download folder and unpack the ZIP archive.
+
+## Install the Extension
+
 <Tabs>
   <TabItem value="chrome" label="Chrome">
 
-#### Open the extensions page.
+#### 1. Open the extension page in your browser.
+
+<Tabs>
+  <TabItem value="chrome-settings" label="Using Settings">
+
+![Step 1 - Chrome: Extension Settings](../../../../static/img/extension/chrome1.png)
+
+  </TabItem>
+  <TabItem value="chrome-menu" label="Using Menu Bar">
+
+![Step 1 - Chrome: Extension Menu Bar](../../../../static/img/extension/chrome2.png)
+
+  </TabItem>
+  <TabItem value="chrome-url" label="Using URL">
+
+Type `chrome://extensions` into the searchbar to open the extension page.
+
+  </TabItem>
+</Tabs>
+
+#### 2. Enable the browser's developer mode.
+
+![Step 2 - Chrome: Enable Developer Mode](../../../../static/img/extension/chrome3.png)
 
-Option 1:  
- ![Step 1a - Install Browser Extension Chrome](../../../../static/img/extension/chrome1.png)
+#### 3. Load the extension from the folder.
 
-Option 2:  
- ![Step 1b - Install Browser Extension Chrome](../../../../static/img/extension/chrome2.png)
+Click _Load unpacked_ pointing to the extracted ZIP archive of the extension.<br/>
+Alternatively, drag and drop the zip file onto the extensions page.
 
-Option 3:  
- in the URL bar of the browser, type `chrome://extensions`
+![Step 3 - Chrome: Load Extension](../../../../static/img/extension/chrome4.png)
```

Comment:
if it is in static, you can load it with this instead:

```suggestion
![Step 3 - Chrome: Load Extension](/img/extension/chrome4.png)
```

Reply:
Good catch. I didn't touch the links at all. We might have to teach the creator about docs.